src/scenes/home/getInvolved/volunteer/volunteer.css

```diff
@@ -0,0 +1,101 @@
+.outerAlt {
+  background-color: #e3e3e3;
+  width: 100%;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  align-content: center;
+  justify-content: center;
+  margin: 5vw;
+}
+
+.half {
+  width: 50%;
+}
+
+.photo {
+  max-width: 500px;
+  height: auto;
+}
+
+.button {
+  display: block;
+  margin: 0 auto;
+  text-align: center;
+}
+
+ul {
```

Comment:
Please no styles applied to elements. We don't have style splitting - any styles applied to elements get applied globally. It happens a few other times in the PR, so please just skim it for the culprits - I'll stop commenting on this issue going forward.